source/includes/indexes/single-field.rb

```diff
@@ -0,0 +1,40 @@
+require 'mongo'
+
+# Replace the placeholders with your credentials
+uri = "<connection string>"
+
+# Sets the server_api field of the options object to Stable API version 1
+options = { server_api: { version: "1" }}
+
+# Creates a new client and connect to the server
+client = Mongo::Client.new(uri, options)
+
+# start-sample-data
+database = client.use('sample_mflix')
+collection = database[:movies]
+# end-sample-data
+
+# start-index-single
+# Creates an index on the "title" field
+index = collection.indexes.create_one({ title: 1 })
```

Comment:
You're assigning `index` to the result of the index creation command, but then never using the variable.

Reply:
If the return value isn't needed just do:

```ruby
collection.indexes.create_one({ title: 1 })
```